app/templates/account/profile.html

```diff
@@ -0,0 +1,22 @@
+{% extends 'layouts/base.html' %}
+{% import 'macros/form_macros.html' as f %}
+
+{% block content %}
+	<div> 
+		{{user.first_name}}
+	</div>
+	<div>
+		{{user.last_name}}
+	</div>
+	<div>
+		{{user.email}}
+	</div>
+	<div>
+		{{role.name}}
```

Comment:
since we won't be passing the `role` variable into the template anymore, this should be `user.role.name`